services/athena/pom.xml

```diff
@@ -61,5 +61,10 @@
             <artifactId>http-auth-aws</artifactId>
             <version>${awsjavasdk.version}</version>
         </dependency>
+        <dependency>
```

Comment:
Can we add `retries-spi` to `services` pom.xml(shared by all services) instead?

Reply:
This change along reduced the complexity of this change a lot, thanks for point this out @zoewangg!